### DISCUSSION TITLE
Add language dataset with M.I.B.

### DISCUSSION THREAD

**Discussion by zrakoplovac** [Mr-MIBonk/M.I.B._More-Incredible-Bash#114] [General]
Hello, how to add more languages with this toolbox (Croatian for example in MHI2)?

I have language datasets in .txt file, but don't know where to put that in M.I.B. solution? Can you explain that in detail? Is it possible at all?

An one more question, how to remove specific FEC with M.I.B., not just illegal ones? Thank you in advance! Cheers 🍻🍻

**Comment by harman-f** [comment]
share that TXT and also your FW information.

Best would be LOG file from M.I.B after you have run a backup.

**Reply by zrakoplovac** [reply]
[lang text.txt](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/10758224/lang.text.txt)
here is .txt
Current FW is this
![image](https://user-images.githubusercontent.com/123552213/219429120-ff174329-d050-4084-b9fd-4a28eac6175b.jpeg)
Car isn't near me so I can't make log.
How do I remove specific FEC? :)

**Reply by harman-f** [reply]
looks like someone copied pieces out of M.I.B
There is a MAX language function included in M.I.B.
It can enable all languages supported by the FW - not able to add new once.

M.I.B. -> Multimedia System -> Languages 

AUG22 supports:
de_DE,en_GB,en_SA,fr_FR,it_IT,es_ES,pt_PT,pt_BR,nl_NL,ru_RU,pl_PL,cs_CZ,es_MX,en_US,sv_SE,tr_TR,hu_HU,no_NO,ar_SA

**Reply by zrakoplovac** [reply]
> looks like someone copied pieces out of M.I.B
> 
> There is a MAX language function included in M.I.B.
> 
> It can enable all languages supported by the FW - not able to add new once.
> 
> 
> 
> M.I.B. -> Multimedia System -> Languages 
> 
> 
> 
> AUG22 supports:
> 
> de_DE,en_GB,en_SA,fr_FR,it_IT,es_ES,pt_PT,pt_BR,nl_NL,ru_RU,pl_PL,cs_CZ,es_MX,en_US,sv_SE,tr_TR,hu_HU,no_NO,ar_SA

Probably...
Yes, I found that menu, but I need Croatian language, if it doesn't support, it doesn't matter...Thank you on detailed reply!

Can I remove some FECs without unistalling everything? Even the valid ones?

**Reply by harman-f** [reply]
FECs: not with M.I.B, only manually. But why would you?

**Reply by zrakoplovac** [reply]
> FECs: not with M.I.B, only manually. But why would you?

Manually, but how exactly?

Because, I added FEC for app connect, and car doesn't have usb socket 8W0035736 to communicate with phone... So, in other words, it isn't working, and it is in menu, but doesn't do anything...
How can I remove that specific FEC? Sorry for bothering you 😅

**Reply by harman-f** [reply]
Just add the proper USB port ;)

**Comment by ivanzelenika95** [comment]
Pozdrav ,

Jesi li se snašao i uspio ikako momče ? , i ja želim Hrvatski jezik u svom Passat-u , brat ima a ja ne. 
Obojica imamo virtual cocpit, molim te odgovori ako si uspio :)
